torchmetrics/image/rase.py

```diff
@@ -0,0 +1,88 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Dict, Optional
+
+import torch
+from torch import Tensor
+
+from torchmetrics.functional.image.rase import _rase_compute, _rase_update
+from torchmetrics.metric import Metric
+
+
+class RelativeAverageSpectralError(Metric):
+    """Computes Relative Average Spectral Error (RASE) (RelativeAverageSpectralError_).
+
+    Args:
+        preds: Deformed image
+        target: Ground truth image
+        window_size: Sliding window used for rmse calculation
+        return_rmse_map: An indication whether
+
+    Return:
+        Relative Average Spectral Error (RASE)
+
+    Example:
+        >>> from torchmetrics import RelativeAverageSpectralError
+        >>> g = torch.manual_seed(22)
+        >>> preds = torch.rand(4, 3, 16, 16)
+        >>> target = torch.rand(4, 3, 16, 16)
+        >>> rase = RelativeAverageSpectralError()
+        >>> rase(preds, target)
+        tensor(8091.1338)
+
+    Raises:
+        ValueError: If ``window_size`` is not a positive integer.
+    """
+
+    total_images: Tensor
+    rmse_map: Tensor = None
+    target_sum: Tensor = None
+    higher_is_better = False
```

Comment:
it seem differentiable to me?
```suggestion
    higher_is_better: bool = False
    is_differentiable: bool = True
```